### PR TITLE
Isocommonref

### PR DESCRIPTION
**Pull request by gdenfield**
Created spikesSiliconProbesV2 file for common reference subtraction and updated detect schema to handle new spike detection method (SiliconProbesV2) which subtracts common reference before running detection. These changes have been tested on the backup database in conjunction with files from other, simultaneous pull request on common ref branch

### CODE REVIEW

File: processing/spikesSiliconProbesV2.m [hunk 1]
```diff
@@ -0,0 +1,20 @@
+function [channels, artifacts] = spikesSiliconProbesV2(sourceFile, spikesFile)
+% Spike detection callback for silicon probes.
+% AE 2011-10-26
+
+
+channels = 1 : 32;
+
+% Extract the common reference - mean across channels at each timepoint
+extract_common_referenceSP(sourceFile);
```
Comment (peabody124) [inline processing/spikesSiliconProbesV2.m]: this file is missing, right?

Comment (peabody124) [inline processing/spikesSiliconProbesV2.m]: also i think you should be able to pass your fixes back to extract_common_reference.m. the only difference between calling it for utah arrays versus these depth probes are the set of channels, which you can pass into the method.

Comment (gdenfield) [inline processing/spikesSiliconProbesV2.m]: Ok I updated those files and pushed the changes. 
extract_common_reference is updated to reflect *SP but that's in the ephys-preprocessing repository, and the spikesCb functions for Utah and SiliconProbesV2 both pass the number of channels as an argument

**Comment by gdenfield** [comment]
Yes, thanks for pointing out the oversight - should be good to go now

**Comment by gdenfield** [comment]
Does everything look in order?

**Comment by gdenfield** [comment]
@aecker Can you merge this and make sure everything looks good after those last changes that were suggested?

**Pull request merged**